chapter2/tests/t19/terraform/prod/prod.tfvars

```diff
@@ -0,0 +1,41 @@
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ env=specific configuration variables                                                                             │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+environment = "prod"
+
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ nginx configuration variables                                                                                    │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+nginx = {
```

Comment:
you should set paths for volume here
i.e.
```hcl
nginx = {
  ...
  volumes = [{
     host="/path"
     container="/tmp/path"
  }]
}
```

something like this.

Reply:
and that should be a relative path, not abs.

Reply:
> you should set paths for volume here i.e.

@dmitry-mightydevops
Done

Reply:
> and that should be a relative path, not abs.

@dmitry-mightydevops
Done